app/assets/javascripts/project.js

```diff
@@ -0,0 +1,64 @@
+var ProjectDashboard = {
+  init: function() {
+    $("#one_year_project, #six_months_project, #three_months_monthly_project, #six_months_monthly_project, #one_year_monthly_project").hide();
+    $('input[name="options_project"]').change( function() {
+      console.log("hiiiiii");
+      ProjectDashboard.update_chart($(this).val());
+    })
+    $('input[name="radio_project"]').change( function() {
+      ProjectDashboard.update_chart($(this).val());
+    })
+  },
+  update_chart: function(months) {
+    if(months == '3' && $('input[name="radio_project"]').filter(':checked').val() == 'weekly') {
```

Comment:
This function is too long and repetitive in general.  It should be broken into at least 2 methods.  First to determine the selection and the second to handle the showing and hiding of components.  Pass whatever you need for the appropriate show into this new method